exercises/intro/intro2.rs

```diff
@@ -2,8 +2,6 @@
 // Make the code print a greeting to the world.
 // Execute `rustlings hint intro2` or use the `hint` watch subcommand for a hint.
 
-// I AM NOT DONE
-
 fn main() {
-    println!("Hello {}!");
+    println!("Hello {}!", "world");
```

Comment:
I noticed that when using single variables you can inline them into the {}
```suggestion
    let world = "world";
    println!("Hello {world}!");
```

```suggestion
    println!("Hello {world}!", world = "world");
```